archive/frames/serializers.py

```diff
@@ -87,7 +88,7 @@ class Meta:
         # }
 
     def create(self, validated_data):
-        version_data = validated_data.pop('version_set')
+        version_data = validated_data.pop('version_set') if 'version_set' in validated_data else {}
```

Comment:
We need to handle the case that we may create a Frame that doesn't yet have a version set - this is when we create a thumbnail whose frame has not yet been created.